car-service/src/main/java/xws/team16/carservice/model/MyImage.java

```diff
@@ -0,0 +1,28 @@
+package xws.team16.carservice.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.sql.Blob;
+
+@Getter @Setter
+@Entity
+public class MyImage {
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "info")
+    private String info;
+
+    @Column(name = "type")
+    private String type;
+
+    @Lob
+    @Column(name = "iamge", columnDefinition="BLOB")
```

Comment:
I mean, this really isn't an issue, but you misspelled "image".

Reply:
Thank you for taking a thorough look, will be fixed right away!